easybuild/framework/easyconfig/parser.py

```diff
@@ -63,7 +63,10 @@ def fetch_parameters_from_easyconfig(rawtxt, params):
     """
     param_values = []
     for param in params:
-        regex = re.compile(r"^\s*%s\s*=\s*(?P<param>\S.*?)\s*$" % param, re.M)
+        if eb_format == '.yeb':
+            regex = re.compile(r"^\s*%s\s*: \s*(?P<param>\S.*?)\s*$" % param, re.M)
+        else:
+            regex = re.compile(r"^\s*%s\s*=\s*(?P<param>\S.*?)\s*$" % param, re.M)
```

Comment:
these are now *very* close to each other, basically it's only `=` vs `:`

also, I think in YAML `=` can also be used?

so, rather than making this regex specific to the format in which the easyconfig is written in, I would just make it a tiny bit more generic, and accept both `=` and `:` as 'assignment' operators, in a single regex:

```
regex = re.compile(r"^\s*%s\s*[=:]\s*(?P<param>\S.*?)\s*$" % param, re.M)
```

also, when touching an existing function like `fetch_parameters_from_easyconfig`, check whether there's a unit test for it, and:

i) if there is, enhance it to also check for the new behaviour
ii) if not, add one, and try to cover all things it should be able to deal with